MeetupPlatform.Api/Features/Meetups/GetReplyComments/Filter/ConfigurationQuery/ReplyCommentFilterHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace MeetupPlatform.Api.Features.Meetups.GetReplyComments.Filter.ConfigurationQuery;
+
+using MeetupPlatform.Api.Domain.Comments;
+using MeetupPlatform.Api.Features.Meetups.GetReplyComments.Filter.FilterSettings;
+using System.Linq;
+
+public static class ReplyCommentFilterHelper
+{
+    public static IQueryable<ReplyComment> GetReplyCommentsFilteredByFilterSettings(
+        this IQueryable<ReplyComment> replyCommentsQuery,
+        ReplyCommentFilterSettings replyCommentFilterSettings)
+        => replyCommentsQuery
+            .SortReplyCommentsByDescending()
+            .GetPaginatedReplyCommentsBySettings(replyCommentFilterSettings);
+
+    private static IQueryable<ReplyComment> SortReplyCommentsByDescending(this IQueryable<ReplyComment> replyCommentsQuery)
+        => replyCommentsQuery.OrderByDescending(replyComment => replyComment.Posted);
```

Comment:
perhaps there should be a choice between sorting in ascending order and sorting in descending order

Reply:
No. Please, read business logic of task.